py/client/pydeephaven/session.py

```diff
@@ -124,6 +191,13 @@ def app_service(self):
 
         return self._app_service
 
+    @property
+    def config_service(self):
+        if not self._config_service:
+            self._config_service = ConfigService(self)
+
+        return self._config_service
+
     def make_ticket(self, ticket_no=None):
```

Comment:
private?
The client API needs to look at all of the methods and confirm if they should be public or not.

I _think_ that the tickets are not exposed to users in the Go client.  Should they be in python?  ... again, something for another PR.

Reply:
see the reason given above.